src/show/components/Navbar/index.js

```diff
@@ -1,63 +1,175 @@
-import { BiSearch } from 'react-icons/bi';
 import { FiUnlock } from 'react-icons/fi';
 import { TbStairsUp } from 'react-icons/tb';
 import { GiTreeGrowth } from 'react-icons/gi';
 import { useNavigate } from 'react-router-dom';
 import { MdOutlineExplore } from 'react-icons/md';
 import { IoConstructOutline } from 'react-icons/io5';
+import { BsFillPersonLinesFill } from 'react-icons/bs';
 
-import { Button } from '../';
+import { Fragment, useState, useEffect } from 'react';
```

Comment:
This should be grouped with the others above.

Reply:
ok :(